src/source/examples/get-ad-v5-pagination-example.rst

```diff
@@ -0,0 +1,51 @@
+.. code-block:: javascript
+
+    1st page is always fetched by applying known query params to the url. The response
+    will indicate if there are additional pages to fetch. The nextPageToken field
```

Comment:
`The response will indicate if there are additional pages to fetch` - how? You explain later on this page, but perhaps already here?

Reply:
Ooh this is Damir's stuff, also don't think it's exposed yet (dangling example) but will fix